--- /usr/bin/raspi-config.orig	2025-09-02 12:59:22.179236681 +0200
+++ /usr/bin/raspi-config	2025-09-02 13:06:43.404133245 +0200
@@ -3484,7 +3484,7 @@
 
   # modify command line
   if ! grep -q "overlayroot=tmpfs" $CMDLINE ; then
-    sed -i $CMDLINE -e "s/^/overlayroot=tmpfs /"
+    sed -i $CMDLINE -e "s/^/overlayroot=tmpfs:recurse=0 /"
   fi
 
   if [ "$BOOTRO" = "yes" ] ; then
@@ -3507,7 +3507,7 @@
   fi
 
   # modify command line
-  sed -i $CMDLINE -e "s/\(.*\)overlayroot=tmpfs \(.*\)/\1\2/"
+  sed -i $CMDLINE -e "s/\(.*\)overlayroot=tmpfs:recurse=0 \(.*\)/\1\2/"
 
   if [ "$BOOTRO" = "yes" ] ; then
     if ! mount -o remount,ro /boot${FIRMWARE} 2>/dev/null ; then
